src/styles/blocks/stars.scss

```
@@ -0,0 +1,21 @@
+.stars {
+  display: flex;
+
+  &__star {
+    width: 16px;
+    height: 16px;
+    background-image: url(../images/star.svg);
+    background-repeat: no-repeat;
+    background-position: center;
```

can be more simplify
```suggestion
    background: url(../images/star.svg) no-repeat center;
```